handbook/growth.md

```diff
@@ -99,6 +99,7 @@ The following table lists the Growth group's rituals, frequency, and Directly Re
 | Enrich Salesforce leads       | Weekly | Follow the Salesforce lead enrichment process every Friday.    | Drew Baker        |
 | Outside contributions | Weekly | Check pull requests for outside contributions every Monday. | Drew Baker|
 | Weekly article       | Weekly | Publish an article and promote it on social media. | Drew Baker|
+| Missed Demo Follow up | Weekly | Email all leads who missed a scheduled demo | Andrew Bare
```

Comment:
You are missing a pipe (|) at the end of this row to close out the last cell in the line.